src/index.scss

```diff
@@ -0,0 +1,55 @@
+@import url('https://fonts.googleapis.com/css2?family=Varela+Round&display=swap');
+
+* {
+  box-sizing: border-box;
+}
+
+#root {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+body {
+  font-family: 'Varela Round', sans-serif;
+  background-color: #e4f5e4;
+  display: flex;
+  justify-content: center;
+  height: 100vh;
+  color: #453d4a;
+}
+
+.btn {
+  border-radius: 50%;
+  border: none;
+  width: 170px;
+  padding: 8px;
+  background-color: #856a94;
```

Comment:
I can suggest creating sass variables (https://sass-lang.com/documentation/variables) like the following
`$error-color: #ff0202;` and use it here ;)

Reply:
Fixed.
Variables used
`$bg-color: #e4f5e4;`